src/main/io/osd.c

```diff
@@ -1146,6 +1146,53 @@ static bool osdDrawSingleElement(uint8_t item)
     return true;
 }
 
+#ifdef USE_OSD_STICK_OVERLAY
+static void osdDrawStockOverlayAxis(uint8_t xpos, uint8_t ypos)
+{
+
+    for (uint8_t x = 0; x < OSD_STICK_OVERLAY_WIDTH; x++) {
+        for (uint8_t y = 0; y < OSD_STICK_OVERLAY_HEIGHT; y++) {
+            // draw the axes, vertical and horizonal
+            if (x == ((OSD_STICK_OVERLAY_WIDTH-1)/2)) {
+                displayWriteChar(osdDisplayPort, xpos+x, ypos+y, osdConfig()->overlay_vertical_char);
+            }
+            if (y == ((OSD_STICK_OVERLAY_HEIGHT-1)/2)) {
+                displayWriteChar(osdDisplayPort, xpos+x, ypos+y, osdConfig()->overlay_horizontal_char);
+            }
+            if ((x == ((OSD_STICK_OVERLAY_WIDTH-1)/2)) && (y == (OSD_STICK_OVERLAY_HEIGHT-1)/2)) {
+                displayWriteChar(osdDisplayPort, xpos+x, ypos+y, osdConfig()->overlay_cross_char);
+            }
+        }
+    }
+}
+
+static void osdDrawStickOverlayAxisItem(osd_items_e osd_item)
+{
+    osdDrawStockOverlayAxis(OSD_X(osdConfig()->item_pos[osd_item]),
+                            OSD_Y(osdConfig()->item_pos[osd_item]));
+}
+
+static void osdDrawStickOverlayPos(osd_items_e osd_item, uint8_t xpos, uint8_t ypos)
+{
+
+    uint8_t elemPosX = OSD_X(osdConfig()->item_pos[osd_item]);
+    uint8_t elemPosY = OSD_Y(osdConfig()->item_pos[osd_item]);
+
+    displayWriteChar(osdDisplayPort, elemPosX+xpos, elemPosY+ypos, osdConfig()->overlay_cursor_char);
+}
+
+static void osdDrawStickOverlayCursor(osd_items_e osd_item, rc_alias_e vertical_channel, rc_alias_e horizontal_channel)
+{
+    uint8_t horizontal = (constrain(rcData[horizontal_channel], PWM_RANGE_MIN, PWM_RANGE_MAX) - PWM_RANGE_MIN) * 100 / (PWM_RANGE_MAX - PWM_RANGE_MIN);
+    uint8_t vertical = 100 - (constrain(rcData[vertical_channel], PWM_RANGE_MIN, PWM_RANGE_MAX) - PWM_RANGE_MIN) * 100 / (PWM_RANGE_MAX - PWM_RANGE_MIN);
+
+    uint8_t x_pos = ((uint16_t)OSD_STICK_OVERLAY_WIDTH * horizontal) / 100;
+    uint8_t y_pos = ((uint16_t)OSD_STICK_OVERLAY_HEIGHT * vertical) / 100;
+
```

Comment:
```
x_pos = scale_range(constrain(rcData[horizontal_channel], PWM_RANGE_MIN, PWM_RANGE_MAX), PWM_RANGE_MIN, PWM_RANGE_MAX, -OSD_STICK_OVERLAY_WIDTH / 2 , OSD_STICK_OVERLAY_WIDTH / 2))
```

Reply:
Also this code ignores `midRc`, which may cause invalid position with centered sticks.

Reply:
Given the resolution, I doubt that will be a problem.